src/app/share/views/SharedLinksView/SharedView.tsx

```diff
@@ -766,19 +768,55 @@ function SharedView(props: SharedViewProps): JSX.Element {
     <div className="h-4 w-20 rounded bg-gray-5" />,
   ];
 
-  const emptyState = (
-    <div className="h-full w-full p-8">
-      <div className="flex h-full flex-col items-center justify-center pb-20">
-        <div className="pointer-events-none mx-auto mb-10 w-max">
-          <Users size={80} weight="thin" />
-        </div>
-        <div className="pointer-events-none text-center">
-          <p className="mb-1 block text-2xl font-medium text-gray-100">{translate('shared-links.empty-state.title')}</p>
-          <p className="block max-w-xs text-lg text-gray-60">{translate('shared-links.empty-state.subtitle')}</p>
+  const emptyState = {
```

Comment:
Extract it to a separate component in order to not add more unnecessary code in the view.